spec/itunes/volume_spec.rb

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+require 'spec_helper'
+require 'itunes-client'
+
+include Itunes
+
+describe Volume do
+  let(:volume) { Itunes::Player.volume }
+  let(:level) { Fixnum }
```

Comment:
Sorry, `20` was a bad example.

`Volume.up`, `Volume.down` methods calls `execute_script` with script path and real level value.
So I think we should check if an instance of Fixnum is given or not.
How do you think this example?

```ruby
describe '.up' do
  it 'calls execute_script with up.scpt and volume value' do
    expect(Volume).to receive(:execute_script).with('volume/up.scpt', an_instance_of(Fixnum))
    Volume.up
  end
end
```